mycroft/skills/core.py

```diff
@@ -1049,7 +1049,7 @@ def __stop_timeout():
                 self.bus.emit(Message("mycroft.stop.handled",
                                       {"by": "skill:"+str(self.skill_id)}))
             timer.cancel()
-        except:
+        except:  # noqa
```

Comment:
Maybe use `except Exception:` instead of bare except. This will catch KeyboardInterrupt and other things.

Reply:
I'm fine with that on this one.  I'll leave it alone in the except on line 1093.

Reply:
Done!